src/components/ChampionCard.jsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Image, Badge, Text, Center, Button, Link } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+
+function ChampionCard({ champion }) {
+  const { name, image, tags } = champion;
+
+  return (
+    <Box
+      maxW='sm'
+      borderWidth='2px'
+      borderRadius='lg'
+      overflow='hidden'
+      boxShadow='md'
+      m='1rem'
+      mb='1rem'
+      _hover={{
+        backgroundColor: 'blue.100',
+        boxShadow: '0 0 10px rgba(0, 0, 0, 0.3)',
+        transition: 'box-shadow 0.3s ease-in-out',
+      }}
+    >
+      <Center h='80%'>
+        <Image src={image} alt={name} objectFit='contain' h='100%' w='100%' />
+      </Center>
+
+      <Box p='2' justifyContent='center'>
+        <Text mt='0' fontWeight='bold' fontSize='l' lineHeight='tight' color='blue.500'>
+          {name}
+        </Text>
+
+        <Box d='flex' alignItems='baseline' justifyContent='center'>
```

Comment:
Tu devrais pouvoir utiliser
```suggestion
        <Flex align='baseline' justify='center'>
```